src/plugins/order_stash.coffee

```diff
@@ -0,0 +1,343 @@
+###
+Plugin which prompts the user to save his/her order
+###
+class OrderStash
+  CTA_READMORE_TEXT = 'Αποθήκευση της παραγγελίας'
+
+  settings = window.sa_plugins.settings
+  context = -> window.sa_plugins.order_stash
+
+  asset_url = (source) -> "#{settings.url.base}/assets/#{source}"
```

Comment:
This one `"@@base/assets/#{source}" ` wouldn't be by default replaced with the base url specified in settings?
Am i wrong?